primer.md

```diff
@@ -235,6 +236,54 @@ general, this is true for all CloudEvents attributes as well.
 This section provides additional background and design points related to some of
 the CloudEvent attributes.
 
+### String Values
+
+Some of the CloudEvent attributes are defined to be of type `String`, and
+all data types used by CloudEvent attribute are required to have a
+string-encoding. This makes for an easy serialization to even the most
+basic formats and transports that might only have a string representation of
+attributes.
+
+Even with that model, it may still be possible to use certain
+characters that could be problematic in some cases. The CloudEvents
+specification does not attempt to further restrict the valid characters used
+for attribute values so as to allow for the specification to be used in
+situations where those potentially problematic characters are desired.
+
+However, this should not be interpreted as encouragement for producers to
+push the boundaries of what characters should be used just for the sake of
+edge-testing implementations. CloudEvents is striving to help with
+interoperability while still being flexible enough to be used in unique
+situations. So, CloudEvent producers are encouraged to only use values that
+have the best chance of being properly processed by all possible formats
+and transports that their CloudEvents might be used with.
```

Comment:
For SaaS publishers, their CloudEvents may end up being used with any format or transport. It'd be good to know the minimum one can rely on. ASCII? URL encoding?

Reply:
I thought about that too, but then I noticed that define Strings as `Sequence of printable Unicode characters`
which to me removed a fair about of the reasons for needing to URL encode stuff. Plus I didn't think we'd want to get into when, or when not, to encode Strings since then we'd need a mechanism by which consumers would have to know when they're encoded.  That's when I decided "just don't be dumb" was good enough.

But if someone has any ideas...